site/en/blog/longer-esw-lifetimes/index.md

```diff
@@ -16,6 +17,10 @@ alt: >
   A picture of an hourglass.
 ---
 
+{% Aside %}
+This post has been updated since we originally published it in January. We've added text indicating that calls to extension APIs reset the service worker's timeout clock, and clarified exactly when service worker timeouts will ocurr.
```

Comment:
Hopefully this is the US spelling too:

```suggestion
This post has been updated since we originally published it in January. We've added text indicating that calls to extension APIs reset the service worker's timeout clock, and clarified exactly when service worker timeouts will occur.
```